.github/workflows/cross-maven.yml

```diff
@@ -70,3 +70,53 @@ jobs:
         run: ./server/mvnw clean install -f core/pom.xml -DskipTests -DskipITs -Dformat.formatter.goal=validate -Dformat.imports.goal=check -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
       - name: Maven build Debezium Server
         run: ./server/mvnw clean install -fae -f server/pom.xml -Passembly -Dformat.formatter.goal=validate -Dformat.imports.goal=check -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 -DskipNonCore
+
+  validate-distribution:
```

Comment:
I wonder if a different job is needed as it executes yet another build. Can't the test be part of `build step?

Reply:
I wanted to have a different job to use the strategy.matrix, so we can execute the same job to cover other different sinks / configurations with little effort.

Reply:
I agree with Mario, if we can reuse the build across jobs since there is this _build_ dependency defined, I think that's reasonable to reduce the overall execution time on CI.

Reply:
@mfvitale @Naros PR updated to reuse the distribution from the Build job. 
I confirmed that the validation job now takes a few seconds when before it used to take some minutes.